Formula/w/woob.rb

```diff
@@ -118,20 +118,11 @@ class Woob < Formula
     sha256 "d0570876c61ab9e520d776c38acbbb5b05a776d3f9ff98a5c8fd5162a444cf19"
   end
 
-  def python3
-    "python3.12"
-  end
-
   def install
     virtualenv_install_with_resources
-
-    site_packages = Language::Python.site_packages(python3)
-    pth_contents = "import site; site.addsitedir('#{libexec/site_packages}')\n"
-    (prefix/site_packages/"homebrew-woob.pth").write pth_contents
```

Comment:
The problem with this is that it pollutes the global environment with all resources, e.g.
```console
❯ /opt/homebrew/bin/python3.12 -c 'import lxml; print(lxml.__file__)'
/opt/homebrew/Cellar/woob/3.6_3/libexec/lib/python3.12/site-packages/lxml/__init__.py
```

This can cause issues due to multiple copies of python packages.